src/Backoffice/Form/MyAccount/ChangeEmailDataMapper.php

```diff
@@ -26,17 +26,17 @@ public function mapDataToForms($viewData, Traversable $forms): void
             return;
         }
 
+        $email = $viewData->email;
```

Comment:
There's no need for this variable, just use `$viewData->email` directly on line 39.